src/phantom.js

```diff
@@ -219,12 +224,32 @@ export default class Phantom {
      */
     exit() {
         clearInterval(this.heartBeatId);
-        this.execute('phantom', 'invokeMethod', ['exit']);
+        return this.execute('phantom', 'invokeMethod', ['exit']);
+    }
+
+    /**
+     * Clean up and force kill this process
+     */
+    kill(errmsg = 'Phantom process was killed') {
```

Comment:
It doesn't make sense to throw an error if the user deliberately/externally kills the process.